src/app/pages/storage/modules/pool-manager/components/pool-manager-wizard/pool-manager-wizard.component.ts

```diff
@@ -97,8 +94,10 @@ export class PoolManagerWizardComponent implements OnInit, OnDestroy {
     this.addVdevsStore.pool$.pipe(
       filter(Boolean),
       tap((pool) => {
-        this.existingPool = _.cloneDeep(pool);
-        this.cdr.markForCheck();
+        if (pool) {
```

Comment:
It looks like unnecessary check due to `filter(Boolean)` above. Is there are cases when it was triggered without a pool?